sal_interfaces/MTM1M3/MTM1M3_Commands.xml

```diff
@@ -302,17 +288,21 @@
   <SALCommand>
     <Subsystem>MTM1M3</Subsystem>
     <EFDB_Topic>MTM1M3_command_clearActiveOpticForces</EFDB_Topic>
-    <item>
-      <EFDB_Name>clearActiveOpticForces</EFDB_Name>
-      <Description>Dummy placeholder</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
+    <Description>
+        The clearActiveOpticForces command shall cause the M1M3 support system
+        to zero the forces applied to the system by the applyActiveOpticForces
+        command.
+    </Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTM1M3</Subsystem>
     <EFDB_Topic>MTM1M3_command_positionM1M3</EFDB_Topic>
+    <Description>
+        Change absolute M1M3 position. Position the mirror with relatively to
+        reference position specified in configuration file. New mirror position
```

Comment:
Position the mirror **relative to the reference** position specified in **the** configuration file